docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLogMeanVarianceFixZero.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.NormalizingTransformer;
+
+namespace Samples.Dynamic
+{
+    public class NormalizeLogMeanVarianceFixZero
+    {
+        public static void Example()
```

Comment:
You have two independent cases in `Example(...)`. Could you split them into two files?

Reply:
I have followed the same pattern as the `MeanVariance` normalizer and `LogMeanVariance` normalizer sample. Would you prefer that I split them all into two files? Or should I just split this one? Or keep it like the others for consistency?

---
In reply to: [298257288](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298257288) [](ancestors = 298257288)

Reply:
Separating them is more reader-friendly.